### PR TITLE
MBS-8828: Add support for showing Songkick urls in sidebar

### PR DESCRIPTION
**Pull request by anshuman73**
Fix for https://tickets.metabrainz.org/browse/MBS-8828

### CODE REVIEW

**Comment by gentlecat** [comment]
@brainzbot, add to whitelist.

**Comment by uklauer** [comment]
Please rewrite the commit message so that it contains the ticket ID and describes what is actually done. E.g., “MBS-8828: Show Songkick URLs in sidebar”

File: lib/MusicBrainz/Server/Data/URL.pm [hunk 1]
```diff
@@ -93,6 +93,7 @@ my %URL_SPECIALIZATIONS = (
     'SetlistFM'           => qr{^https?://(?:www.)?setlist.fm/}i,
     'SMDB'                => qr{^https?://(?:www.)?smdb.kb.se/}i,
     'Songfacts'           => qr{^https?://(?:www.)?songfacts.com/}i,
+    'Songkick'            => qr{^https?://(?:www.)?songkick.com/},
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/Data/URL.pm]: All the other regexes use the /i modifier, why not this one?

File: lib/MusicBrainz/Server/Entity/URL/Sidebar.pm [hunk 2]
```diff
@@ -86,7 +86,8 @@ use constant FAVICON_CLASSES => {
     'indiegogo.com'             => 'indiegogo',
     'kickstarter.com'           => 'kickstarter',
     'setlist.fm'                => 'setlistfm',
-    'vimeo.com'                 => 'vimeo'
+    'vimeo.com'                 => 'vimeo',
+    'songkick.com'              => 'songkick'
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/Entity/URL/Sidebar.pm]: Just a note, you can add a trailing comma after the last element in the list, and it'll make additions cleaner in the future (note how it has to remove the vimeo line first to append a comma).

**Comment by anshuman73** [comment]
The last commit was to resolve the merge conflicts due to https://github.com/metabrainz/musicbrainz-server/pull/412

**Pull request merged**